app/controllers/public_pages_controller.rb

```diff
@@ -18,7 +18,11 @@ def source_routing
     redirect_to root_path
   end
 
-  def home; end
+  def home
+    if session[:source] == "full-service"
```

Comment:
If they come in via `/full-service` then happily go to intake, and then they decide to learn more about GYR, will this change mean that when they view the homepage, they get redirected back into intake?

Reply:
I like that!